opensearch/src/test/java/org/opensearch/sql/opensearch/client/OpenSearchNodeClientTest.java

```diff
@@ -280,7 +280,7 @@ void search() {
 
     Iterator<ExprValue> hits = response1.iterator();
     assertTrue(hits.hasNext());
-    assertEquals(exprTupleValue, hits.next());
+    assertEquals(exprTupleValue.tupleValue().get("id"), hits.next().tupleValue().get("id"));
```

Comment:
it was matching the whole object - and failing because the Boolean object wasn't .equal.  This could be removed (I think) if I change the Boolean to a boolean

Reply:
Sorry - I got that wrong.  We are now returning the metafields in the response from OpenSearch.. so they're appearing in the tuple.  This way the spirit of the test remains the same.